server/index.js

```diff
@@ -33,6 +33,7 @@ Promise.all([
     blockchain.subscribe()
 
     app.use('/fish', express.static('../fish_client/public'))
```

Comment:
If we leave the fish ui in the server, is that going to require the AMI adds both the fish type and the new basic asset type to the genesis block?

Reply:
This will eventually be configurable. Probably this week.